src/Types/Conversions/ros/tf/StampedTransform.hpp

```diff
@@ -8,7 +8,7 @@ namespace robot_remote_control {
 namespace RosConversion {
 
     static void convert(const tf::StampedTransform &from, robot_remote_control::Pose *to) {
-        convert(from.header, to->mutable_header());
+        //convert(from.header, to->mutable_header());
```

Comment:
Why is this commented? Since we have no distinction between Pose and StampedPose, I would leave the header in the conversion. Dropping the header purposely removes information that is to be transmitted. I understand that this is converting from transform to pose, but still the header contains information especially about the time that we should not disregard. The library is a transport library and imo should not edit the data, when not necessary. If the header is not used on the receiver side it should be handled there.

Reply:
tf::StampedTransform has no header field http://docs.ros.org/en/hydro/api/tf/html/c++/classtf_1_1StampedTransform.html

but it has a "stamp" field with the time, which should be used now